test/test-matrix-api-r0.3.0.el

```diff
@@ -0,0 +1,106 @@
+(require 'cl-lib)
+(require 'eieio)
+(require 'map)
+(require 'seq)
+
+(require 'a)
+(require 'buttercup)
+(require 'f)
+(require 'json)
+(require 'request)
+
+
+(describe "matrix-client"
+
+  (let ((load-path (list default-directory
+                         (f-parent default-directory)
+                         (f-join default-directory "test"))))
+    ;;  (require 'matrix-api-r0.3.0)
+    (load "matrix-api-r0.3.0")
+    (load "password"))
+
+  (before-all
+    (setq session (matrix-session :user "@matrix-client-el-tester:matrix.org"
```

Comment:
I think it would be nice to supply the username in the password file as well, in case people want to use their own accounts for testing.

Reply:
Great idea.